ci/gdk_build.template.steps.yaml

```diff
@@ -28,19 +28,12 @@ env:
   FASTBUILD_BROKERAGE_PATH: "\\\\fastbuild-brokerage.${CI_ENVIRONMENT}-intinf-eu1.i8e.io\\samba"
 
 steps:
-  - label: "build-and-test-GDK-:windows:-ENGINE_COMMIT_HASH_PLACEHOLDER"
-    command: "powershell ./ci/setup-build-test-gdk.ps1 -target_platform Win64"
-    <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
-    artifact_paths:
-      - "../UnrealEngine/Engine/Programs/AutomationTool/Saved/Logs/*"
-      - "../UnrealEngine/Samples/StarterContent/Saved/Logs/*"
-    env:
-      ENGINE_COMMIT_HASH: "ENGINE_COMMIT_HASH_PLACEHOLDER"
-
-  - label: "build-GDK-:linux:-ENGINE_COMMIT_HASH_PLACEHOLDER"
-    command: "powershell ./ci/setup-build-test-gdk.ps1 -target_platform Linux"
+  - label: "build-ENGINE_COMMIT_HASH_PLACEHOLDER-BUILD_PLATFORM_PLACEHOLDER-BUILD_TARGET_PLACEHOLDER-BUILD_STATE_PLACEHOLDER"
+    command: "powershell ./ci/setup-build-test-gdk.ps1 -target_platform BUILD_PLATFORM_PLACEHOLDER"
     <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
     artifact_paths:
       - "../UnrealEngine/Engine/Programs/AutomationTool/Saved/Logs/*"
     env:
       ENGINE_COMMIT_HASH: "ENGINE_COMMIT_HASH_PLACEHOLDER"
+      BUILD_TARGET: "BUILD_TARGET_PLACEHOLDER"
+      BUILD_STATE: "BUILD_STATE_PLACEHOLDER"
```

Comment:
Shame that Unreal _actually_ uses the term "build state" to refer to debug/release type of configuration 😕

Reply:
Yeah, it definitely doesn't help in terms of clarity 😢